tests/test_suite.rs

```diff
@@ -1,12 +1,43 @@
-use draft::test_draft;
-
-test_draft!("tests/suite/tests/draft4/", {
-    "optional_bignum_0_0",
-    "optional_bignum_2_0",
-});
-test_draft!("tests/suite/tests/draft6/");
-test_draft!("tests/suite/tests/draft7/", {
+use json_schema_test_suite::{json_schema_test_suite, TestCase};
+use jsonschema::{Draft, JSONSchema};
+
+#[json_schema_test_suite("tests/suite", "draft4", {"optional_bignum_0_0", "optional_bignum_2_0"})]
+#[json_schema_test_suite("tests/suite", "draft6")]
+#[json_schema_test_suite("tests/suite", "draft7", {
     "optional_format_idn_hostname_0_11", // https://github.com/Stranger6667/jsonschema-rs/issues/101
     "optional_format_idn_hostname_0_6",  // https://github.com/Stranger6667/jsonschema-rs/issues/101
     "optional_format_idn_hostname_0_7",  // https://github.com/Stranger6667/jsonschema-rs/issues/101
-});
+})]
+fn test_draft(_server_address: &str, test_case: TestCase) {
+    let draft_version = match test_case.draft_version.as_ref() {
+        "draft4" => Draft::Draft4,
+        "draft6" => Draft::Draft6,
+        "draft7" => Draft::Draft7,
+        _ => panic!("Unsupported draft"),
+    };
+
+    let compiled = JSONSchema::compile(&test_case.schema, Some(draft_version)).unwrap();
+
+    let result = compiled.validate(&test_case.instance);
```

Comment:
It seems that checking equivalence of `compiled.validate` and `compiled.is_valid` is missing

Reply:
I feel like ensuring that ensuring
 * if `schema.is_valid(instance)` then `schema.validate(instance)` will have no errors
 * if `!schema.is_valid(instance)` then `schema.validate(instance)` will have at least one error
should not be done into an integration test.

and so I did remove it as it felt like it was an unintentional addition, but this is a low-cost addition, so don't necessarily mind adding it back

Reply:
Hm, initially my thoughts were about keeping those shortcut implementations in sync with the more verbose ones and I felt like this is a convenient place to do so :)